build.gradle

```diff
@@ -286,7 +289,7 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '6.2.0'
+  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '6.5.4-ACA-113'
```

Comment:
This will need updating when the following befta-fw updates are merged, see hmcts/befta-fw#128.